pom.xml

```diff
@@ -117,6 +117,7 @@
         <module>plugin/trino-hive</module>
         <module>plugin/trino-hive-hadoop2</module>
         <module>plugin/trino-iceberg</module>
+        <module>plugin/trino-ignite</module>
```

Comment:
update `core/trino-server/src/main/provisio/presto.xml` too, otherwise the plugin is not part of the release